src/Native/System.Net.Security.Native/pal_gssapi.cpp

```diff
@@ -102,9 +102,11 @@ extern "C" uint32_t NetSecurityNative_DisplayStatus(uint32_t* minorStatus,
     assert(outBuffer != nullptr);
 
     int statusType = isGssMechCode ? GSS_C_MECH_CODE : GSS_C_GSS_CODE;
+    uint32_t messageContent;
     GssBuffer gssBuffer {.length = 0, .value = nullptr};
     uint32_t majorStatus =
-        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, nullptr, &gssBuffer);
+        gss_display_status(minorStatus, statusValue, statusType, GSS_C_NO_OID, &messageContent, &gssBuffer);
+    (void)messageContent; // unused
```

Comment:
Interesting... was this necessary?  I'm surprised the compiler warned in this case.  Is that because it's attributed to be an out value on OS X or something?